### ISSUE TITLE
MTL-Tensorflow.ipynb bug

### ISSUE THREAD

**Issue by EnnengYang** [hosseinshn/Basic-Multi-task-Learning#1]
`        if epoch % 1 == 0:
            print('Epoch: %d, Train Cost: %5.3f ' % (epoch, epoch_cost))
            dv_cost = sess.run(cost, feed_dict={X: X_valid, Y1: Y1_valid, Y2: Y2_valid, kp: 0.5})
            dv_cost1 = sess.run(cost1, feed_dict={X: X_valid, Y1: Y1_valid, Y2: Y2_valid, kp: 0.5})
            cost1D.append(dv_cost1)
            dv_cost2 = sess.run(cost2, feed_dict={X: X_valid, Y1: Y1_valid, Y2: Y2_valid, kp: 0.5})
            cost2D.append(dv_cost2)
`
Dropout should not be used in the evaluation,  so should you change it to the following?
`        if epoch % 1 == 0:
            print('Epoch: %d, Train Cost: %5.3f ' % (epoch, epoch_cost))
            dv_cost = sess.run(cost, feed_dict={X: X_valid, Y1: Y1_valid, Y2: Y2_valid, **kp: 1**})
            dv_cost1 = sess.run(cost1, feed_dict={X: X_valid, Y1: Y1_valid, Y2: Y2_valid, **kp: 1**})
            cost1D.append(dv_cost1)
            dv_cost2 = sess.run(cost2, feed_dict={X: X_valid, Y1: Y1_valid, Y2: Y2_valid, **kp: 1**})
            cost2D.append(dv_cost2)
`
